### ISSUE TITLE
Missing increment overflow warning using interval sets

### ISSUE THREAD

**Issue by Robotechnic** [goblint/analyzer#1798]
It seems like Goblint doesn't detect the integer overflow in this program:

```c
int count_int_int;
void main() {
  for (;; count_int_int++)
    ;
}
```

I don't know if it is normal or not, as this is an empty infinite loop.
